src/server/lock_manager/deadlock.rs

```diff
@@ -319,6 +329,9 @@ impl Display for Task {
             ),
             Task::DetectRpc { .. } => write!(f, "Detect Rpc"),
             Task::ChangeRole(role) => write!(f, "ChangeRole {{ role: {:?} }}", role),
+            Task::ChangeTTL(ttl) => write!(f, "ChangeTTL {{ ttl: {:?} }}", ttl),
```

Comment:
```suggestion
            Task::ChangeTTL(ttl) => write!(f, "ChangeTTL { ttl: {:?} }", ttl),
```

Reply:
we need `{{` here to print out  `{`, so `Task::ChangeTTL` print something like `ChangeTTL { ttl: 100ms }`.